pkg/controller/appinst_api.go

```diff
@@ -2266,6 +2305,7 @@ func setPortFQDNPrefix(port *edgeproto.InstPort, objs []runtime.Object) {
 			if err != nil {
 				return
 			}
+			// http proto should not have prefixes
```

Comment:
Hm, not sure I understand your question. But let me clarify that comment:
```
			// in case of HTTP ports, the proto will never match
			// any of the kubernetes service ports which are only
			// TCP/UDP, which is what we want because http ports
			// are routed via ingress and use the App's URI, and
			// do not need fqdn prefixes which are used to route
			// to LBs.
```